src/pks/flow/constitutive_relations/thaw_depth/activelayer_average_temp_evaluator.cc

```diff
@@ -17,11 +17,11 @@ ActiveLayerAverageTempEvaluator::ActiveLayerAverageTempEvaluator(Teuchos::Parame
     : SecondaryVariableFieldEvaluator(plist)
 {
   domain_ = Keys::getDomain(my_key_);
-  auto pos = domain_.find_last_of('_');
+  auto pos = domain_.find_last_of(':');
   int col_id = std::stoi(domain_.substr(pos+1, domain_.size()));
```

Comment:
This kind of thing can become:

`int col_id = Keys::getDomainSetIndex<int>(domain_);`

Reply:
done